plaso/parsers/sqlite_plugins/windows_timeline.py

```diff
@@ -0,0 +1,225 @@
+          '[PlatformDeviceId] TEXT, [CreatedInCloud] DATETIME, [StartTime] '
+          'DATETIME, [EndTime] DATETIME, [LastModifiedOnClient] DATETIME, '
+          '[GroupAppActivityId] TEXT, [ClipboardPayload] BLOB, [EnterpriseId] '
+          'TEXT, [OriginalPayload] BLOB, [OriginalLastModifiedOnClient] '
+          'DATETIME, [ETag] INT NOT NULL)'),
+      'ActivityAssetCache': (
+          'CREATE TABLE [ActivityAssetCache]([ResourceId] INTEGER PRIMARY KEY '
+          'AUTOINCREMENT NOT NULL, [AppId] TEXT NOT NULL, [AssetHash] TEXT '
+          'NOT NULL, [TimeToLive] DATETIME NOT NULL, [AssetUri] TEXT, '
+          '[AssetId] TEXT, [AssetKey] TEXT, [Contents] BLOB)'),
+      'ActivityOperation': (
+          'CREATE TABLE [ActivityOperation]([OperationOrder] INTEGER PRIMARY '
+          'KEY ASC NOT NULL, [Id] GUID NOT NULL, [OperationType] INT NOT '
+          'NULL, [AppId] TEXT NOT NULL, [PackageIdHash] TEXT, [AppActivityId] '
+          'TEXT, [ActivityType] INT NOT NULL, [ParentActivityId] GUID, [Tag] '
+          'TEXT, [Group] TEXT, [MatchId] TEXT, [LastModifiedTime] DATETIME '
+          'NOT NULL, [ExpirationTime] DATETIME, [Payload] BLOB, [Priority] '
+          'INT, [CreatedTime] DATETIME, [Attachments] TEXT, '
+          '[PlatformDeviceId] TEXT, [CreatedInCloud] DATETIME, [StartTime] '
+          'DATETIME NOT NULL, [EndTime] DATETIME, [LastModifiedOnClient] '
+          'DATETIME NOT NULL, [CorrelationVector] TEXT, [GroupAppActivityId] '
+          'TEXT, [ClipboardPayload] BLOB, [EnterpriseId] TEXT, '
+          '[OriginalPayload] BLOB, [OriginalLastModifiedOnClient] DATETIME, '
+          '[ETag] INT NOT NULL)'),
+      'Activity_PackageId': (
+          'CREATE TABLE [Activity_PackageId]([ActivityId] GUID NOT NULL, '
+          '[Platform] TEXT NOT NULL, [PackageName] TEXT NOT NULL, '
+          '[ExpirationTime] DATETIME NOT NULL)'),
+      'AppSettings': (
+          'CREATE TABLE [AppSettings]([AppId] TEXT PRIMARY KEY NOT NULL, '
+          '[SettingsPropertyBag] BLOB, [AppTitle] TEXT, [Logo4141] TEXT)'),
+      'ManualSequence': (
+          'CREATE TABLE [ManualSequence]([Key] TEXT PRIMARY KEY NOT NULL, '
+          '[Value] INT NOT NULL)'),
+      'Metadata': (
+          'CREATE TABLE [Metadata]([Key] TEXT PRIMARY KEY NOT NULL, [Value] '
+          'TEXT)')}]
+
+  def ParseGenericRow(
+      self, parser_mediator, query, row, **unused_kwargs):
+    """Parses a generic windows timeline row.
+
+      Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+          and other components, such as storage and dfvfs.
+      query (str): query that created the row.
+      row (sqlite3.Row): row.
+    """
+    query_hash = hash(query)
+
+    event_data = WindowsTimelineGenericEventData()
+
+    # Payload is JSON serialized as binary data in a BLOB field, with the text
+    # encoded as UTF-8.
+    payload_json_bytes = bytes(self._GetRowValue(query_hash, row, 'Payload'))
```

Comment:
Turns out `_GetRowValue` returns a `buffer` type for `Payload` - I think this is the best way to deal with that.

Reply:
Looks good